HHS/Head-Start-TTADP
# Adjust spacing, make scrollbar for my reports always visible and other tweaks.

**Description of change**
Adjusts spacing for the navigation pages to better match the design. The scrollbar for the  main activity reports table is always visible now to avoid confusion. Other minor tweaks to match with the design.

**How to test**
- pull changes
- verify scrollbar is visible, things work as before, but match the design better

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/0

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
